BaseClasses.py

```diff
@@ -1389,6 +1390,14 @@ def write_option(option_key: str, option_obj: Options.AssembleOptions) -> None:
 
             AutoWorld.call_all(self.multiworld, "write_spoiler", outfile)
 
+            precollected_items = [f'{item.name} ({self.multiworld.get_player_name(item.player)})'
+                                  if self.multiworld.players > 1
+                                  else item.name
+                                  for item in chain.from_iterable(self.multiworld.precollected_items.values())]
+            if precollected_items:
+                outfile.write('\n\nPrecollected Items:\n\n')
+                outfile.write('\n'.join([item for item in precollected_items]))
```

Comment:
i think i'd prefer wrapping this in a counter and showing the count with the item rather than just repeating the item `count` times.

Reply:
i was thinking something like
```
Five Rings: 1 (PoryGone_SA2B1)
Shield: 5 (PoryGone_SA2B)
```

Reply:
I would think it's probably slightly preferable to just list them each individually and match stylistically, I can also see some weird edge cases around item names with numbers in them if you try to add a quantity

Reply:
those would still be separated by the colon. ```
Rupees (500): 2 (Player3)
```

Reply:
just checked and quite a lot do. whether that makes the information difficult to parse is a different question. most notable ones would be stardew and raft which have items like `Resource Pack: 2000 Money` or `Resource Pack: 5 Plastic`